src/datalayer/reducers/auth.reducer.js

```diff
@@ -10,6 +10,13 @@ export const INITIAL_STATE = {
 
 export default (state = INITIAL_STATE, action) => {
   switch (action.type) {
+    case authAction.LOAD_AUTH: {
+      return {
+        ...state,
+        loggedIn: null,
+      };
+    }
+
```

Comment:
Why need to do this?

Reply:
Remember pressing back button at Home screen and reopen the app and the app will be infinite loop?
This is the problem

Reply:
Ok

Reply:
Vì nó check sự thay đổi props loggedIn, có sự thay đổi thì mới đi tiếp (componentDidUpdate)
Nếu như cũ thì khi thoát ra rồi vào lại, cái prop đó vẫn giữ nguyên
Cho nên mình cần phải reset prop này